kibana-reports/server/routes/report.ts

```diff
@@ -203,6 +213,7 @@ export default function (router: IRouter) {
           `Failed to generate report from reportDefinition id ${reportDefinitionId} : ${error}`
         );
         logger.error(error);
+        addToMetric('report', 'create', checkErrorType(error), report);
```

Comment:
Is there a chance `report` will be undefined here?

Reply:
yes, but it doesn't matter. `report` will only become undefined when the backend call to retrieve the metadata fails. Then we won't update any business metric in that case, since we don't know it's a csv or pdf/png.